framework/test/jest/unit/specs/controller/application_state.spec.js

```diff
@@ -93,7 +93,7 @@ describe('Application State', () => {
 		describe('when there is an error', () => {
 			// Arrange
 			const newState = {
-				broadhash: 'xxx',
+				prevotedConfirmedUptoHeight: 0,
```

Comment:
Can you tell why this is throwing an error?

Reply:
In this case we are checking what happens when there is an error thrown by publishing an event using the `channel`